Practice Areas/Engineering/code_review_guidelines.md

```diff
@@ -12,12 +12,20 @@ Before sending your code to a VSP team for review, your code should meet all of
 ## Guidelines
 
 * **Review lifecycle**
-    * Before sending code to VSP team, have it reviewed by your team (see **First review** below).
-    * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate to Leah and Patrick.
+    * Before sending code to VSP team have it reviewed by your team (see **First review** below) using a [Draft Pull Request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).  
+      * This prevents [code owners](https://help.github.com/en/articles/about-code-owners) from being notified before pull request is ready for review.
+      * [Learn more about draft pull requests](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+    * After having your pull request reviewed by your team you can click the *Ready for review* button in the merge section of your pull request.
+      * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate via your DSVA product owner.
+      * Please **do not post in Slack channels requesting VSP code reviews** unless it has been more then 24 hours (business days).
```

Comment:
```suggestion
      * **Do not post in Slack channels requesting VSP code reviews** unless it has been more than 24 hours since requesting review on GitHub.
```